doc/classes/Timer.xml

```diff
@@ -37,14 +38,17 @@
 			<argument index="0" name="paused" type="bool">
 			</argument>
 			<description>
-				Set whether the timer is paused or not. A paused timer will be inactive until it is unpaused again.
+				Pause the timer. If [code]paused[/code] is [code]true[/code], the timer will not process until it is started or unpaused again, even if [code]start[/code] is called.
+				Default: [code]false[/code]
 			</description>
 		</method>
 		<method name="start">
 			<return type="void">
 			</return>
 			<description>
 				Start the Timer.
+				This also resets the timeout to [code]wait_time[/code].
+				If the timer is paused, the timer will not resume automatically.
```

Comment:
I'm not sure this is getting across the point, which is that `start()` doesn't resume a paused timer. Maybe:

Note: this method will not resume a paused timer. See [method set_paused].